apps/web/src/components/Icon/Icon.stories.tsx

```diff
@@ -0,0 +1,46 @@
+import Icon from './Icon';
+import { ReactComponent as RedditIcon } from '../../assets/redditIcon.svg';
+import { ReactComponent as UserIcon } from '../../assets/userIcon.svg';
+import { ReactComponent as HomeIcon } from '../../assets/homeIcon.svg';
+
+export default {
+  title: 'Components/Icon',
+  component: Icon,
+};
+
+interface IconArgs {
+  /**
+   * child icon SVG component passed to the Icon component
+   */
+  children: JSX.Element;
+  /**
+   * global css styles applied to all of our Icon components
+   */
+  style?: React.CSSProperties;
+}
+
+const sharedStyles = {
+  width: '100%',
+  height: '100%',
+  margin: '5px',
+};
+
+const Template = (args: IconArgs): JSX.Element => <Icon {...args}></Icon>;
+
+export const Reddit = (args: IconArgs) => <Template {...args} />;
```

Comment:
When creating a story I like to always have a 'Default' story that includes all of the props that will be the same across my different components. So for this I'd pass an svg in the `children` prop and `sharedStyles to `style`. Then for very other story I can spread in the `Default.args` object, saving me some lines. So it would look something like:

```tsx
export const Default = (args: IconArgs) => <Template {...args} />;
Default.args = {
children: <DefaultIcon color='#ff2800' />,
style: sharedStyles,
}

export const Reddit = (args: IconArgs) => <Template {...args />;
Reddit.args = {
  ...Default.args,
  children: <RedditIcon color="ff2800" />,
}
```